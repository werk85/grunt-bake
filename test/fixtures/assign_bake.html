<!DOCTYPE html>
<html>
	<head></head>
	<body>
		<!--(bake includes/second.html title="Hello World!" _assign="foo")-->

		<p>
			{{foo}}
		</p>
	</body>
</html>