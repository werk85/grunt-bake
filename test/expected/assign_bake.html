<!DOCTYPE html>
<html>
	<head></head>
	<body>

		<p>
			<span>Hello World!</span>
		</p>
	</body>
</html>